charts/dbildungs-iam-server/values.yaml

```diff
@@ -189,6 +188,9 @@ backend:
     stepUpTimeoutEnabled: 'false'
     maxOldSpaceSize: 3584
 
+redis: 
+  host: dbildungs-iam-redis-master.spsh.svc.cluster.local
```

Comment:
Can't we just use dbildungs-iam-redis-master here since they're in the same namespace?